### PR TITLE
Mm platform

### PR DESCRIPTION
**Pull request by karsev**
New hw platform

### CODE REVIEW

File: NFC_EEPROM/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You will also need to supply the driver for the EEPROM. This example is known to
 Clone the repository containing the collection of examples:
 
 ```
-git clone https://github.com/ARMmbed/mbed-os-example-nfc.git
+git clone https://github.com/vince-zeng/mbed-os-example-nfc.git
```
Comment (pan-) [inline NFC_EEPROM/README.md]: No reason to change that link.

File: NFC_EEPROM/README.md [hunk 2]
```diff
@@ -54,6 +54,11 @@ Run the build:
 mbed compile -t <ARM | GCC_ARM> -m <YOUR_TARGET>
 ```
 
+For NXP K64F, run the following command:
```
Comment (pan-) [inline NFC_EEPROM/README.md]: Not required, it is documented above. We try to not favour a platform.

File: NFC_EEPROM/TARGET_MM/PinNames.h [hunk 3]
```diff
@@ -0,0 +1,343 @@
+/*
```
Comment (pan-) [inline NFC_EEPROM/TARGET_MM/PinNames.h]: It doesn't seem right to have that file in that PR. `PinNames.h` should be provided by the mbed os target.

File: NFC_EEPROM/TARGET_MM/device.h [hunk 4]
```diff
@@ -0,0 +1,23 @@
+// The 'features' section in 'target.json' is now used to create the device's hardware preprocessor switches.
```
Comment (pan-) [inline NFC_EEPROM/TARGET_MM/device.h]: Same as above; you should add the target to mbed-os; not here.

File: NFC_EEPROM/source/main.cpp [hunk 5]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+#include "mbed.h"
```
Comment (pan-) [inline NFC_EEPROM/source/main.cpp]: No, avoid to include mbed.h directly as it pulls out a lot of includes and pollute the global namespace with its `using namespace std` an `using namespace mbed statements`. 

If you need something specific. Include it and import it in the local namespace (if your in a cpp file) if you want shorter names. 

```c++ 
#include "Drivers/DigitalIOut.h"

using mbed::DigitalOut;
```

File: NFC_EEPROM/source/main.cpp [hunk 6]
```diff
@@ -37,6 +37,16 @@ using mbed::nfc::ndef::common::span_from_cstr;
 /* URL that will be written into the tag */
 const char url_string[] = "mbed.com";
 
+#ifdef TARGET_MM
```
Comment (pan-) [inline NFC_EEPROM/source/main.cpp]: No, this has nothing to do in the main file. Hide these details in specific target code.

File: NFC_EEPROM/mbed_app.json [hunk 7]
```diff
@@ -1,8 +1,13 @@
 {
+  "macros": ["NFC_I2C_SDA=D14","NFC_I2C_SCL=D15","NFC_FIELD_DETECTION=D5"
```
Comment (pan-) [inline NFC_EEPROM/mbed_app.json]: That is specific to a certain config; please add it to the K64F configuration and not the global configuration.

File: NFC_EEPROM/mbed_app_mm.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (pan-) [inline NFC_EEPROM/mbed_app_mm.json]: Please remove that file.

File: NFC_EEPROM/source/target/TARGET_NT3H2111/eeprom_driver.lib [hunk 9]
```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-nfc-nt3h2111
```
Comment (pan-) [inline NFC_EEPROM/source/target/TARGET_NT3H2111/eeprom_driver.lib]: Could you specify the hash to load ?

File: NFC_EEPROM/custom_targets.json [hunk 10]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (pan-) [inline NFC_EEPROM/custom_targets.json]: Forgot that one; it should be removed from this PR as well.

File: NFC_EEPROM/README.md (hunk 1 above)
Comment (karsev) [reply NFC_EEPROM/README.md]: Hmm, didn't mean to change this

File: NFC_EEPROM/TARGET_MM/PinNames.h (hunk 3 above)
Comment (karsev) [reply NFC_EEPROM/TARGET_MM/PinNames.h]: this is custom target not meant to be in mbed-os

**Comment by karsev** [comment]
I will remove this PR. This wasn't the right place for it.

File: NFC_EEPROM/TARGET_MM/PinNames.h (hunk 3 above)
Comment (pan-) [reply NFC_EEPROM/TARGET_MM/PinNames.h]: It has not place here.

**Pull request closed without merging**